projects/demo/src/app/app.component.html

```diff
@@ -16,3 +16,5 @@
 >
   <router-outlet></router-outlet>
 </sdg-navigation>
+
+<sdg-footer [footer]="footer"></sdg-footer>
```

Comment:
On pourrait wrapper dans un élément footer pour améliorer la sémantique et l'accessibilité
```suggestion
<footer>
  <sdg-footer [footer]="footer" />
</footer>
```